sdk-extensions/autoconfigure/build.gradle.kts

```diff
@@ -97,16 +88,6 @@ testing {
         }
       }
     }
-    val testInitializeRegistersGlobal by registering(JvmTestSuite::class) {
-      targets {
-        all {
-          testTask {
-            environment("OTEL_TRACES_EXPORTER", "none")
-            environment("OTEL_METRICS_EXPORTER", "none")
-          }
-        }
-      }
-    }
```

Comment:
These testsuites were split out because they left behind resources that causes issues with other tests. Being able to merge them into the other test suites was a decent proxy for confirming that all tests were properly cleaning up resources.